en/02_Developer_Guides/02_Controllers/04_Redirection.md

```diff
@@ -11,13 +11,12 @@ HTTP header.
 
 **app/src/Page.php**
 
-
 ```php
 $this->redirect($this->Link('goherenow'));
 // redirect to Page::goherenow(), i.e on the contact-us page this will redirect to /contact-us/goherenow/
 
-$this->redirect('goherenow/');
-// redirect to the URL on yoursite.com/goherenow/. (note the trailing slash)
+$this->redirect('goherenow');
+// redirect to the URL on yoursite.com/goherenow/
```

Comment:
would this depend on the trailing slash config whether it has or doesn't have the trailing slash?

Reply:
Regardless of trailing slash config, you can include or omit a trailing slash in your call to `redirect()`.

Reply:
This has been changed in https://github.com/silverstripe/developer-docs/pull/188 to move the slash to be a _leading_ slash, which I think was the original intention (i.e. it goes to `yoursite.com/goherenow` as opposed to `yoursite.com/some/existing/path/goherenow`)